client/coral-admin/src/routes/Moderation/components/Comment.js

```diff
@@ -127,7 +128,7 @@ class Comment extends React.Component {
               </span>
 
               <span className={styles.created}>
```

Comment:
Would be nice if this element also became `<time className={styles.created} datetime={comment.created_at} title={comment.created_at}>` - more semantic and absolute time could also be viewed as a tooltip

Reply:
done